packages/vue-vanilla/tests/unit/complex/ObjectRenderer.spec.ts

```diff
@@ -49,4 +66,14 @@ describe('ObjectRenderer.vue', () => {
     const inputs = wrapper.findAll('input');
     expect(inputs.length).to.equal(2);
   });
+
+  it('renders a string input with an asterisk', () => {
+    const wrapper = mountJsonForms('a', schemaRequired, uischema);
+    expect(wrapper.find('label span.asterisk').exists()).to.be.true;
+  });
+
+  it('renders a string input but hide asterisk', () => {
+    const wrapper = mountJsonForms('a', schemaRequired, uischemaRequired);
+    expect(wrapper.find('label span.asterisk').exists()).to.be.false;
+  });
```

Comment:
I think these tests fit better to `StringControlRenderer.spec.ts` because they are not related to the object renderer itself.

Reply:
but isn't the "required" feature a part of the object?

https://json-schema.org/understanding-json-schema/reference/object#required

Reply:
It is conceptually, however the rendering of the "required" is not part of the object renderer. Therefore it should be tested in the respective renderers instead.